poc/tenant-controller/pkg/controllers/tenants/controller.go

```diff
@@ -63,7 +70,21 @@ func (c *Controller) Run(ctx context.Context) error {
 func (c *Controller) createTenant(obj *tenantsapi.Tenant) {
 	glog.Info("createTenant: %#v", obj)
 	syncRBACForTenantCRD(obj)
-	// TODO create namespace
+
+	// TODO Add later ... sanity checks to ensure namespaces being requested are valid and not already assigned to another tenant
+	c.tenantsList = make(map[string]tenantsapi.Tenant)
```

Comment:
The map should be initialized in `NewController`. But I don't see the need of using this map, shall we add that later when we need it?

Reply:
Ifwe decide to remove tenantsList, then I can remove this also, lets wait and see what we do with that first.